include/pinocchio/parsers/graph/geometries.hpp

```diff
@@ -0,0 +1,128 @@
+//
+// Copyright (c) 2025 INRIA
+//
+
+#ifndef __pinocchio_parsers_geometries_graph_hpp__
+#define __pinocchio_parsers_geometries_graph_hpp__
+
+#include "pinocchio/spatial/se3.hpp"
+
+namespace pinocchio
+{
+  namespace graph
+  {
+    // Kind of possible geometry
+    enum GEOM_TYPE
```

Comment:
```suggestion
    enum struct GeomType
```
Enum are in CamelCase.
We can use enum struct to avoid enum to leaking inside the namespace (we will have to allway do `GeomType::VISUAL`)